src/components/PeopleFilters.tsx

```diff
@@ -1,16 +1,103 @@
+import { useSearchParams } from 'react-router-dom';
+import { SexEnum } from '../types/enums';
+import cn from 'classnames';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('century') || [];
+  const allCenturies = [16, 17, 18, 19, 20];
+
+  function handleSexFiltering(
+    event: React.MouseEvent<HTMLAnchorElement>,
+    sex: string | null,
+  ) {
+    event.preventDefault();
+
+    const params = new URLSearchParams(searchParams);
+
+    if (sex) {
+      params.set('sex', sex);
+    } else {
+      params.delete('sex');
+    }
+
+    setSearchParams(params);
+  }
+
+  function handleQueryChange(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
It is important to follow the same way of creating functions, fix it everywhere
```suggestion
  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```